packages/core-contracts/src/contracts/FleetCommander.sol

```diff
@@ -475,6 +495,23 @@ contract FleetCommander is
         _updateRebalanceCooldown(newCooldown);
     }
 
+    /// @inheritdoc IFleetCommander
+    function updateWithdrawalFee(
+        Percentage newFee
+    ) external onlyGovernor whenNotPaused {
+        _updateWithdrawalFee(newFee);
+    }
+
+    /// @inheritdoc IFleetCommander
+    function getWithdrawalFee()
```

Comment:
Not needed, it is already inherited from WithdrawalFee. Also function should be external if left here